Core/PSPLoaders.cpp

```diff
@@ -322,5 +323,14 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 	pspFileSystem.Mount("umd0:", fs);
 
 	std::string finalName = ms_path + file + extension;
+
+	std::string homebrewName = PSP_CoreParameter().fileToStart;
```

Comment:
PBP seems to be just generic container, so it can store just the elf, like eboot.pbp on game disc, or be a disc on it's own, homebrew pbp file structure reminds me of update files pbp :|, maybe it's based on it.